en/setup/dev_env_windows_cygwin.md

```diff
@@ -192,10 +184,10 @@ This section describes how to setup the Cygwin toolchain manually yourself while
 
 1. Write up or copy the **batch scripts** [`run-console.bat`](https://github.com/MaEtUgR/PX4Toolchain/blob/master/run-console.bat) and [`setup-environment-variables.bat`](https://github.com/MaEtUgR/PX4Toolchain/blob/master/toolchain/setup-environment-variables.bat). 
    
-   The reason to start all the development tools through the prepared batch scripts is they preconfigure the starting program to use the local, portable Cygwin environment inside the toolchain's folder. This is done by always first calling the script [**setup-environment-variables.bat**](https://github.com/MaEtUgR/PX4Toolchain/blob/master/toolchain/setup-environment-variables.bat) and the desired application like the console after that.
+   The reason to start all the development tools through the prepared batch script is they preconfigure the starting program to use the local, portable Cygwin environment inside the toolchain's folder. 
```

Comment:
Thanks. The docs toolchain doesn't care about this. I do slightly, but I wouldn't address it properly (ie via tooling) because that would be an unnecessary barrier to contribution.